mne/realtime/mock_lsl_stream.py

```diff
@@ -86,5 +86,5 @@ def _initiate_stream(self):
             mysample = self._raw[:, counter][0].ravel().tolist()
             # now send it and wait for a bit
             outlet.push_sample(mysample)
-            counter += 1
+            counter = 0 if counter == raw.last_samp else counter + 1
```

Comment:
I get this:

```py
Traceback (most recent call last):
  File "/home/mainak/anaconda2/envs/mne/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/home/mainak/anaconda2/envs/mne/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/home/mainak/Desktop/projects/github_repos/mne-python/mne/realtime/mock_lsl_stream.py", line 89, in _initiate_stream
    counter = 0 if counter == raw.last_samp else counter + 1
NameError: name 'raw' is not defined
```

can you add a test so that this does not happen?

Reply:
oops, was a copy error from testing

Reply:
it was me not realizing that i wasn't using self